HOWTO-make-a-release.md

```diff
@@ -0,0 +1,348 @@
+
+## Updating the website
+
+Push the website changes you made earlier to the OpenSSL website repo.  When
+you do this, the website will get updated and a script to flush the Akamai CDN
+cache will be run.  You can look at things on www-origin.openssl.org; the
+CDN-hosted www.openssl.org should only be a few minutes delayed.
+
+# Post-publishing tasks
+
+## Check the website
+
+Verify that the release notes, which are built from the CHANGES file in the
+release, have been updated.  This is done automatically by the commit-hook,
+but if you see a problem, try the following steps on `dev.openssl.org`:
+
+    cd /var/www/openssl
+    sudo -u openssl -H make relupd
+    sudo -u openssl ./bin/purge-one-hour
+
+Wait for a while for the Akamai flush to work (normally within a few minutes).
+Have a look at the website and news announcement at:
+
+-   <https://www.openssl.org/>
+-   <https://www.openssl.org/news/>
+
+Check the download page has updated properly:
+
+-   <https://www.openssl.org/source/>
+
+Check the notes look sensible at:
+
+-   <https://www.openssl.org/news/newslog.html>
+
+Also check the notes here:
+
+-   <https://www.openssl.org/news/openssl-1.0.2-notes.html>
+-   <https://www.openssl.org/news/openssl-1.1.0-notes.html>
+-   <https://www.openssl.org/news/openssl-1.1.1-notes.html>
+
+## Send the announcement mail
+
+Send out the announcements.  Generic release announcement messages will be
+created automatically by the build script and the commands you need to use
+to send them were displayed when you executed do-release.pl above.
+These should normally be sent from the openssl account.  These are sent to
+openssl-users, openssl-project, and openssl-announce.
+
+If do-release.pl was used with `--move` be sure to move the announcement
+text files away from the staging directory after they have been sent.  This
+is done as follows (with VERSION replaced with the version of OpenSSL to
+announce):
+
+        sudo -u openssl \
+            mv ~openssl/dist/new/openssl-VERSION.txt.asc ~openssl/dist/old
+
+Send out the Security Advisory if there is one.  Copy the file to the
+openssl user home directory, and then do the following:
+
+        sudo -u openssl gpg -u 8B3D79F5 --clearsign secadv_FILENAME
+        sudo -u openssl mutt -s "OpenSSL Security Advisory" \
+                openssl-project openssl-users openssl-announce \
+                <~openssl/secadv_FILENAME.txt.asc
+
+Approve the openssl-announce email.  Go to
+<https://mta.openssl.org/mailman/admindb/openssl-announce>
+and approve the messages.
+The administration password needed for approval is held in /opt/mailman/README
+on mta.openssl.org
+
+Check the mailing list messages have arrived.
+
+## Unfreeze the source repository.
+
+    ssh openssl-git@git.openssl.org unfreeze openssl
+
+## Security fixes
+
+If this release includes security fixes with a CVE then you should inform
+MITRE about them.  See the instructions at the top of cvepool.txt in omc.
```

Comment:
cvepool.txt is a private file only accessible to OMC. We might want to change the location of the instructions - or perhaps just split out the bits related to informing MITRE.

Reply:
Another PR to figure that out, maybe?

Reply:
Sure